packages/experiments/src/components/Card/CardItem/CardItem.types.ts

```diff
@@ -0,0 +1,30 @@
+import { IComponentStyles, IHTMLSlot, IComponent, IStyleableComponentProps } from '../../../Foundation';
+
+export type ICardItemComponent = IComponent<ICardItemProps, ICardItemTokens, ICardItemStyles>;
+
+export interface ICardItemSlots {
+  root?: IHTMLSlot;
+}
+
+// These types are redundant with ICardItemComponent but are needed until TS function return widening issue is resolved:
+// https://github.com/Microsoft/TypeScript/issues/241
+// For now, these helper types can be used to provide return type safety when specifying tokens and styles functions.
+export type ICardItemTokenReturnType = ReturnType<Extract<ICardItemComponent['tokens'], Function>>;
+export type ICardItemStylesReturnType = ReturnType<Extract<ICardItemComponent['styles'], Function>>;
+
+export interface ICardItemProps extends ICardItemSlots, IStyleableComponentProps<ICardItemProps, ICardItemTokens, ICardItemStyles> {
+  /**
+   * Defines how to render the CardItem.
+   */
+  as?: string | React.ReactType<ICardItemProps>;
+
+  /**
+   * Defines if the default padding applies to this CardItem or not.
+   * @defaultvalue false
+   */
+  preventPadding?: boolean;
```

Comment:
Hmm, I wondering about this prop in architecturally descending order 😆 
1. Is there a strong reason not to just use a `padding` or `paddingHorizontal` token? It seems that this is used to display images nicely aligned to the Card. Wouldn't it be reasonable to let consumers handle the styling to eliminate padding (a relatively common scenario) rather than introduce a prop to do it?

1. I wonder if this should be a token somehow? It's similar to top level props like `disabled` and `primary` (argument against being a token), but it is also used only in the styles function (argument for being a token?)

1. I'm also not clear on what "default padding" here means because when this prop is TRUE the negative padding is applied. What default padding is being prevented?

1. I'm not sure about the name. Is this an existing prop in OUFR? `disablePadding` seems more consistent with our naming conventions.

Reply:
In order:
1. Not really a strong reason to have this like we this, just taking inspiration from the `preventShrink` prop in `Stack`, but happy to just use a `padding` or `paddingHorizontal` token. Don't really have a strong preference here. Maybe @dzearing, @micahgodbolt and @mikewheaton could share their opinions about this.

2. Definitely should be a token if we decide to go for a `padding` or `paddingHorizontal` approach.

3. Maybe I should improve the comments here but the default padding in the `Card` is `12px` in every direction. So when we prevent this default padding for just a child, we apply a negative margin to make up for this.

4. It's not an existing prop, again, was going of the `preventShrink` prop in `Stack` but happy to take suggestions on naming. `disablePadding` could definitely work.

Reply:
Good point on `preventShrink`, I wish I would have thought to call it out as `disableShrink` earlier. I would still lean towards `disablePadding` though. 😆 

Better yet (per our chat), perhaps `disableChildPadding` to indicate it applies to the children. Any input we can get from others would help, too.